kolibri/core/auth/models.py

```diff
@@ -81,13 +81,14 @@
 
 logger = logging.getLogger(__name__)
 
+# '"optional":True' is obsolete but needed while we keep using an
+# old json_schema_validator version compatible with python 2.7
```

Comment:
Maybe also expand and mention that this needs to be backwards compatible? I'm thinking it would be a good thing given I'm guessing your migration here will stop Django wanting to create more migrations for schema changes?

Reply:
Yep, I had hesitated to do it, but it's good to avoid the same problem again. Expanded